src/test/unit/ntypes/DimensionsTest.cpp

```diff
@@ -71,6 +71,7 @@ TEST_F(DimensionsTest, InvalidDimensions) {
   ASSERT_FALSE(d.isDontcare());
   ASSERT_TRUE(d.isInvalid());
   ASSERT_FALSE(d.isSpecified());
+  //EXPECT_STREQ("[1,0] (Invalid) ", d.toString().c_str());
```

Comment:
if this is a TODO, we should re-enable, 
if the functionality is removed, rm the line

Reply:
> if this is a TODO, we should re-enable,

I could not decide which way to go.  I am thinking that once Cereal is in place and we no longer use this in the reverse direction I should enable it.  I will mark this as TODO:

Hmmm, since I am going to have a lot of TODO's related to Cereal maybe I should mark them as such so that after it gets installed everywhere I can find those and fix them up.

Reply:
:+1: //TODO CEREAL , or such ..